kingpin/actors/aws/ecs.py

```diff
@@ -615,7 +668,7 @@ class Service(ECSBaseActor):
     """
 
     all_options = {
-        'state': (STATE, 'present', 'Desired state: present/absent'),
+        'desired_state': (STATE, 'present', 'Desired state: present/absent'),
```

Comment:
Please keep this as `state` ... thats the same verb we use in all of the other state-ensuring actors, so consistency wins here IMHO.